extension/Makefile

```diff
@@ -0,0 +1,62 @@
+PG_CONFIG=pg_config
+
+EXTENSION=timescale_prometheus_support
```

Comment:
why don't we just call this timescale-prometheus?

Reply:
The name is ambiguous, it's difficult to tell if `timescale-prometheus` refers to the connector or the extension. Since the only time the name will ever come up is during debugging, I think that disambiguation is more important than prettiness

Reply:
let's call it `timescale-prometheus-extra` then note also - instead of _